apps/portal/components/pagesComponents/profileScreen/DeleteAccountModal.tsx

```
@@ -181,21 +190,22 @@ const PermanentlyDeleteAccount: FC<IPermanentlyDeleteAccountProps> = ({
   return (
     <div className={styles.permanentlyDeleteAccount}>
       <p>
-        Are you sure you want to delete your Rec account? Deleting your
-        <b> {email}</b> account will delete all your associated data such as
-        your images, videos, folders, and creative spaces, including content you
-        have shared with others. You cannot undo this operation.
+        <Trans
+          values={{ email: email }}
+          i18nKey="page.profile.deleteAccountModal.message"
+          components={{
+            1: <b></b>,
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)